package.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<package packagerversion="1.9.4" version="2.0" xmlns="http://pear.php.net/dtd/package-2.0" xmlns:tasks="http://pear.php.net/dtd/tasks-1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0 http://pear.php.net/dtd/tasks-1.0.xsd http://pear.php.net/dtd/package-2.0 http://pear.php.net/dtd/package-2.0.xsd">
+    <name>datadog_trace</name>
+    <channel>pecl.php.net</channel>
+    <summary>APM and distributed tracing for PHP</summary>
+    <description>The Datadog PHP Tracer brings APM and distributed tracing to PHP.</description>
+    <lead>
+        <name>Sammy Kaye Powers</name>
+        <user>sammyk</user>
+        <email>sammyk@php.net</email>
+        <active>yes</active>
+    </lead>
+    <date>2018-12-14</date>
+    <version>
+        <release>0.8.1</release>
+        <api>0.8.1</api>
+    </version>
+    <stability>
+        <release>beta</release>
+        <api>beta</api>
+    </stability>
+    <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
+    <notes>
+        * Update Symfony 3 and 4 docs #184
+        * Package installation on custom PHP setups lacking conf.d support #188
+    </notes>
+    <contents>
+        <dir name="/">
+            <dir name="src">
+                <dir name="ext">
+                    <file name="compat_zend_string.c" role="src" />
+                    <file name="compat_zend_string.h" role="src" />
+                    <file name="ddtrace.c" role="src" />
+                    <file name="ddtrace.h" role="src" />
+                    <file name="debug.h" role="src" />
+                    <file name="dispatch.c" role="src" />
+                    <file name="dispatch.h" role="src" />
+                    <file name="dispatch_compat.h" role="src" />
+                    <file name="dispatch_compat_php5.c" role="src" />
+                    <file name="dispatch_compat_php5.h" role="src" />
+                    <file name="dispatch_compat_php7.c" role="src" />
+                    <file name="dispatch_setup.c" role="src" />
+                    <file name="version.h" role="src" />
+                </dir>
+            </dir>
+            <dir name="tests">
```

Comment:
Is it a practice to include tests in pear packages? (asking) 😄

Reply:
Yep apparently it is: https://github.com/phpredis/phpredis/blob/develop/package.xml#L113

Reply:
Definitively YES.
Test are useful (at least) for downstream packager